### PR TITLE
Column width not getting changed

### PR DESCRIPTION
**Pull request by Akarshit**
@benfry I have changed the labels to read "`prettyVersion` installed/available". I hope this is what you wanted.
I18n will be done in a separate PR for the whole CM.

### CODE REVIEW

**Comment by benfry** [comment]
Can you re-base this and re-submit?

**Comment by Akarshit** [comment]
@benfry I have re-based this.

File: app/src/processing/app/contrib/Contribution.java [hunk 1]
```diff
@@ -321,4 +321,23 @@ static private String translateCategory(String cat) {
     String cleaned = cat.replaceAll("[\\W]+", "_").toLowerCase();
     return Language.text("contrib.category." + cleaned);
   }
+  
+  String getPrettyVersionShort() {
+
+    String latestVersion = this.getPrettyVersion();
+    if (latestVersion != null && !latestVersion.isEmpty()) {
+      if (latestVersion.endsWith("alpla")) {
```
Comment (benfry) [inline app/src/processing/app/contrib/Contribution.java]: Let's not get into parsing/cleaning what other people have put into the fields. We should contact library authors and have them fix the files (which can be done quickly), not change code that might stick around for years.

File: app/src/processing/app/contrib/ContributionListing.java [hunk 2]
```diff
@@ -526,18 +526,7 @@ String getLatestVersion(Contribution contribution) {
...
-        return ("v" + latestVersion.substring(5, latestVersion.indexOf(','))
-            .trim());
-      else if (latestVersion.toLowerCase().startsWith("v")) // For ketai library
-        return latestVersion;
-      else
-        return ("v" + latestVersion);
-    }
-    else
-      return null;
+    return newestContrib.getPrettyVersionShort();
```
Comment (benfry) [inline app/src/processing/app/contrib/ContributionListing.java]: No need for a whole function here... Just use the `prettyVersion` and set the max size of the field. When the authors see their stuff looks goofy, that's a motivation to fix it.

File: app/src/processing/app/contrib/ContributionManagerDialog.java [hunk 3]
```diff
@@ -79,7 +79,7 @@
 
   public ContributionManagerDialog() {
     myFont = Toolkit.getSansFont(14, Font.PLAIN);
-//    GraphicsEnvironment.getLocalGraphicsEnvironment().registerFont(myFont);
+    GraphicsEnvironment.getLocalGraphicsEnvironment().registerFont(myFont);
```
Comment (benfry) [inline app/src/processing/app/contrib/ContributionManagerDialog.java]: Why is this re-enabled?

File: app/src/processing/app/contrib/StatusPanel.java [hunk 4]
```diff
@@ -174,11 +173,11 @@ public void actionPerformed(ActionEvent e) {
                               .addComponent(updateLabel))
                   .addComponent(updateButton).addComponent(removeButton)));
 
-    layout.linkSize(SwingConstants.HORIZONTAL, installButton, progressBarPanel,
```
Comment (benfry) [inline app/src/processing/app/contrib/StatusPanel.java]: Please don't reformat like this. See the style guidelines (and where that's missing things, see the rest of the code). This change makes things less readable.

File: app/src/processing/app/contrib/StatusPanel.java [hunk 5]
```diff
@@ -234,21 +233,19 @@ public void update(ContributionPanel panel) {
...
-    } else {
+    if (currentVersion == null) {
       currentVersion = "";
     }
 
-    if (updateButton.isEnabled()) {
+    if(updateButton.isEnabled()){
       updateButton.setText(latestVersion);
-    } else {
+    }else{
```
Comment (benfry) [inline app/src/processing/app/contrib/StatusPanel.java]: See style guidelines. Don't change this.

**Comment by benfry** [comment]
This commit is a mess. You've done enough of these that you should be aware of the [style guidelines](https://github.com/processing/processing/wiki/Style-Guidelines) and even if not, simply following the existing layout from the entire rest of the codebase. This is a project with a large number of contributors, and if everyone does whatever they want, the code will be a disaster.

**Comment by Akarshit** [comment]
@benfry I understand what you said. I am closing this and will submit a new PR with the changes.

**Pull request closed without merging**